beta-pipelines/broad/somatic/single_sample/wgs/gdc_genome/GDCWholeGenomeSomaticSingleSample.wdl

```diff
@@ -657,9 +685,19 @@ workflow GDCWholeGenomeSomaticSingleSample {
             bqsr_recal_file = gatk_baserecalibrator.bqsr_recal_file
     }
 
+    String output_bam_prefix = basename(gatk_applybqsr.bam, ".bam")
+
+    call collect_insert_size_metrics {
+        input:
+            input_bam = gatk_applybqsr.bam,
+            output_bam_prefix = "~{output_bam_prefix}.insert_size_histogram.pdf"
```

Comment:
Should the output_bam_prefix include ".insert_size_histogram.pdf"?

Reply:
Good question. Will th resulting file end up with this included twice? ".insert_size_histogram.pdf.insert_size_histogram.pdf"

Reply:
yup! thanks for catching @gbggrant. It doubled the file ext ".insert_size_histogram.pdf.insert_size_histogram.pdf"